src/py/flwr/client/app.py

```diff
@@ -268,6 +297,12 @@ def _load_client_app(_1: str, _2: str) -> ClientApp:
 
         load_client_app_fn = _load_client_app
 
+    if isolate:
+        # Start gRPC server
+        clientappio_servicer, _ = run_clientappio_api_grpc(
+            address=ADDRESS_CLIENTAPPIO_API_GRPC_RERE
+        )
+
```

Comment:
I might be mistaken, but would this not start a server for each client? I would suggest starting one server when the SuperNode starts and reusing that one.